Lesson11/lesson11_3.cpp

```diff
@@ -0,0 +1,48 @@
+// Lesson 11 3
+#include <iostream>
+
+const int COL = 4;
+const int ROW = 4;
+
+void passArrV1(int arr_2d[ROW][COL])
+{
+    std::cout << "Passing array v1: " << std::endl;
+        for(int j = COL - 1; j >= 0 ; j--){
+            for(int i = 0; i < ROW; i++){
+            arr_2d[i][j] = 0;
```

Comment:
why are you doing it?

Reply:
a bit comfortable to read, personally. if i'am understand you correctly.